powerquery-docs/Connectors/DatabricksCloud.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Databricks connector
+description: Provides basic information and prerequisites for using Power Query's Databricks connectors.
+author: George Chow
+
+ms.topic: conceptual
+ms.date: 6/29/2022
+ms.author: George Chow
+LocalizationGroup: reference
+---
+
+# Databricks Cloud
+
+## Summary
+
+| Item | Description |
+| ---- | ----------- |
+| Release State | Azure Databricks: General Availability<br/>Databricks: Preview |
+| Products | Power BI (Datasets)<br/>Power BI (Dataflows) |
+| Authentication Types Supported | Azure Active Directory<br/>Organizational Account<br/>Personal Access Token |
+| | |
+
+>[!Note]
+>Some capabilities may be present in one product but not others due to deployment schedules and host-specific capabilities.
+
+
+## Prerequisites
+
+You requires either an Azure Databricks account or a paid Databricks Cloud account to use this connector. Databricks Community Edition is not supported.
+
+## Capabilities supported
+
+* Import
+* DirectQuery (PowerBI only)
+
+## Connect to Azure Databricks or Databricks Cloud from Power Query Desktop
```

Comment:
same comment in regard to Databricks Cloud.

```suggestion
## Connect to Azure Databricks or Databricks on other cloud providers from Power Query Desktop
```

Reply:
k